internal/bminventory/inventory.go

```diff
@@ -2955,6 +2968,15 @@ func (b *bareMetalInventory) setDefaultUsage(cluster *models.Cluster) {
 	withCredentials := cluster.Platform.Vsphere != nil && cluster.Platform.Vsphere.VCenter != nil && cluster.Platform.Vsphere.Password != nil && cluster.Platform.Vsphere.Username != nil
 	b.setUsage(cluster.Platform.Type != models.PlatformTypeBaremetal, usage.PlatformSelectionUsage, &map[string]interface{}{
 		"platform_type": cluster.Platform.Type, "with_credentials": withCredentials}, usages)
+	if cluster.DiskEncryption != nil {
```

Comment:
since you use it twice, put this in a function and use in both placed. Also please remove the debug print

Reply:
Will Fix it.

Reply:
Fixed.